web-console/src/views/ingestion-view/ingestion-view.tsx

```diff
@@ -1138,6 +1155,12 @@ ORDER BY "rank" DESC, "created_time" DESC`;
                 localStorageKey={LocalStorageKeys.TASKS_REFRESH_RATE}
                 onRefresh={auto => this.taskQueryManager.rerunLastQuery(auto)}
               />
+              {/* Toggle switch to query overload or RDS(RDS network throughput increases 3X on firing the UI) */}
+              <Switch
+                label="Query(Overlord/RDS)"
```

Comment:
SQL would be a more accurate here, since that's how it's referred to everywhere else in the code. RDS is an implementation detail